dev_environment/e2e_test/start_test_servers.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
Isn't this the same file in the parent "dev_environment" directory?  Do we have to duplicate another copy of this?

Reply:
path of web_server.go is different